b2b-sample/apps/business-admin-app/components/sections/sections/settingsSection/idpSection/idpSectionComponent.tsx

```diff
@@ -16,50 +16,47 @@
  * under the License.
  */
 
-import { IdentityProvider, IdentityProviderTemplate, getCallbackUrl } from
+import { IdentityProvider, IdentityProviderTemplate } from
     "@b2bsample/business-admin-app/data-access/data-access-common-models-util";
-import { controllerDecodeCreateIdentityProvider, controllerDecodeListAllIdentityProviders } from
+import { controllerDecodeListAllIdentityProviders } from
     "@b2bsample/business-admin-app/data-access/data-access-controller";
-import { EmptySettingsComponent, SettingsTitleComponent, errorTypeDialog, infoTypeDialog, successTypeDialog } from
+import { EmptySettingsComponent, SettingsTitleComponent, errorTypeDialog, successTypeDialog } from
     "@b2bsample/shared/ui/ui-components";
-import {
-    CopyTextToClipboardCallback, EMPTY_STRING, ENTERPRISE_ID, GOOGLE_ID,
-    checkIfJSONisEmpty, copyTheTextToClipboard, sizeOfJson
-} from
-    "@b2bsample/shared/util/util-common";
 import AppSelectIcon from "@rsuite/icons/AppSelect";
-import CopyIcon from "@rsuite/icons/Copy";
-import InfoRoundIcon from "@rsuite/icons/InfoRound";
-import React, { useCallback, useEffect, useState } from "react";
-import { Avatar, Button, Container, FlexboxGrid, Form, Input, InputGroup, Modal, Panel, Stack, useToaster } from
-    "rsuite";
+import { Session } from "next-auth";
+import { useCallback, useEffect, useState } from "react";
+import { Container, useToaster } from "rsuite";
 import IdentityProviderList from "./otherComponents/identityProviderList";
-import config from "../../../../../../../config.json";
+import IdpCreate from "./otherComponents/idpCreateModal/idpCreate";
+import SelectIdentityProvider from "./otherComponents/selectIdentityProvider";
 import Enterprise from
     // eslint-disable-next-line
     "../../../../../../../libs/business-admin-app/data-access/data-access-common-models-util/src/lib/identityProvider/data/templates/enterprise-identity-provider.json";
 import Google from
     // eslint-disable-next-line    
     "../../../../../../../libs/business-admin-app/data-access/data-access-common-models-util/src/lib/identityProvider/data/templates/google.json";
```

Comment:
The file paths are longer than they should be. Shall we try to reduce the number of directories here? For instance, we do we have `src/lib` inside `data-access-common-models-util`?